File: src/main/java/ru/protei/oxmldoc/common/RowLimitRules.java
package ru.protei.oxmldoc.common;

public enum RowLimitRules {

    /**
     * Ignore limit value, allow to append any number of rows
     */
    IGNORE,

    /**
     * Skip all rows above limit
     */
    SKIP,

    /**
     * Generate error
     */
    ERROR,

    /**
     * Automatically create a new worksheet and proceed append to it
     */
    SPLIT
}

File: src/main/kotlin/ru/protei/oxmldoc/common/RowLimitRules.kt
package ru.protei.oxmldoc.common

enum class RowsLimitRule {

    /**
     * Ignore limit value, allow to append any number of rows
     */
    IGNORE,

    /**
     * Skip all rows above limit
     */
    SKIP,

    /**
     * Generate error
     */
    ERROR,

    /**
     * Automatically create a new worksheet and proceed append to it
     */
    SPLIT
}